rinja/src/lib.rs

```diff
@@ -180,9 +180,11 @@ impl<T: Template + ?Sized> Template for &T {
     const SIZE_HINT: usize = T::SIZE_HINT;
 }
 
-/// Object-safe wrapper trait around [`Template`] implementers
+/// [`dyn`-compatible] wrapper trait around [`Template`] implementors.
 ///
-/// This trades reduced performance (mostly due to writing into `dyn Write`) for object safety.
+/// This trades reduced performance (mostly due to writing into `dyn Write`) for dyn-compatibility.
+///
+/// [`dyn`-compatible]: <https://doc.rust-lang.org/stable/reference/items/traits.html#dyn-compatibility>
```

Comment:
Not sure it's valid markdown.

```suggestion
/// [`dyn`-compatible]: https://doc.rust-lang.org/stable/reference/items/traits.html#dyn-compatibility
```

Reply:
It is. You can even try it in this chat. :)

Reply:
So weird. Normally you use `<>` to linkify a link directly in text, not in definition. Is there any advantage to use this syntax here?

Reply:
When used in inline text, you have to surround a link with `<…>` so it becomes clickable. I always add braces in all places, so it always looks the same. (I think github-compatible markdown makes anything clickable that starts with `http[s]://`.)